src/ast.rs

```diff
@@ -1394,6 +1405,50 @@ where
     }
 }
 
+/// <clone-suffix> ::= [ . <clone-type-identifier> ] [ . <nonnegative number> ]*
```

Comment:
Where did you find this production? I can't find anything about clone-suffix in the ABI document, only in code comments in other demanglers.

Reply:
yay for standards :-/

Thanks again for the fix!

Reply:
Seriously lol, hopefully the abi gets updated soon :sweat_smile:
Yeah of course have a good week :+1: